library/build.gradle

```diff
@@ -3,8 +3,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.2'
```

Comment:
Is updating protobuf in the scope of this PR?

Reply:
i have updated only if not ok make you values